openbb_platform/providers/polygon/openbb_polygon/models/balance_sheet.py

```diff
@@ -20,8 +21,13 @@ class PolygonBalanceSheetQueryParams(BalanceSheetQueryParams):
     Source: https://polygon.io/docs/stocks#!/get_vx_reference_financials
     """
 
-    __alias_dict__ = {"symbol": "ticker", "period": "timeframe"}
+    __alias_dict__ = {"symbol": "ticker"}
 
+    period: Optional[Literal["annual", "quarter", "ttm"]] = Field(
```

Comment:
This doesn't appear to work, all three periods return exactly the same thing:

![Screenshot 2023-10-19 at 11 47 57 AM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/8f036ce5-a892-46c2-a2aa-c9d96b06c23b)

![Screenshot 2023-10-19 at 11 54 16 AM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/634edfec-3b43-40ae-ae46-43acff80fd37)

![Screenshot 2023-10-19 at 11 55 52 AM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/e1fc80ac-564a-48cd-b244-6f3d5f72b644)